docs/contributing/building/environment.md

```diff
@@ -84,47 +92,47 @@ builds (and executes) correctly in both environments.
 * To create an **alpine-tracee-make** container:
 
     ```text
-    $ make -f builder/Makefile.tracee-make alpine-prepare
+    make -f builder/Makefile.tracee-make alpine-prepare
     ```
 
 * To create an **ubuntu-tracee-make** container:
 
     ```text
```

Comment:
why same are `text` and others `console?`

Reply:
Thanks. It's a leftover. Fixing it.